hr_holidays_credit/models/hr_leave_type.py

```diff
@@ -47,27 +47,28 @@ def name_get(self):
 
             extra = None
             if record.allocation_type != 'no' and context_employee_id:
-                if record.virtual_remaining_leaves >= 0:
+                amount = float_round(record.virtual_remaining_leaves, precision_digits=2) or 0.0
+                if amount >= 0 :
+                    if record.request_unit == 'day':
+                        extra = _('%g day available') % amount if abs(amount) <= 1 else _('%g days available') % amount
+                    elif record.request_unit == 'hour':
+                        extra = _('%g hour available') % amount if abs(amount) <= 1 else _('%g hours available') % amount
+                    else:
+                        extra = _('%g %s available') % (amount, record.request_unit)
+
                     if record.allow_credit:
-                        extra = _('%g available + credit')
+                        extra += " " + _('+ credit')
+
+                elif amount < 0:
+                    if record.request_unit == 'day':
+                        extra = _('%g day used in credit') % amount if abs(amount) <= 1 else _('%g days used in credit') % amount
+                    elif record.request_unit == 'hour':
+                        extra = _('%g hour used in credit') % amount if abs(amount) <= 1 else _('%g hours used in credit') % amount
                     else:
-                        extra = _('%g available')
-                    extra = extra % (
-                        float_round(
-                            record.virtual_remaining_leaves,
-                            precision_digits=2
-                        ) or 0.0,
-                    )
-                elif record.allow_credit:
-                    extra = _('%g used in credit') % (
-                        float_round(
-                            -record.virtual_remaining_leaves,
-                            precision_digits=2
-                        ) or 0.0,
-                    )
+                        extra = _('%g %s used in credit') % (amount, record.request_unit)
 
             if extra:
-                record_name = _('%(name)s (%(extra)s)') % {
+                record_name = '%(name)s (%(extra)s)' % {
```

Comment:
Why did you remove translation here? RTL languages kind-a have that backwards

Reply:
To save white bears :angel:  to me it's useless because it mean you have strings like : "Legal Leaves 2021 (20 days available)" in your po file, wich is very unlikely

Reply:
That's not how translation works in this case. Template is getting localized, not formatted string

Reply:
I don't doubt what you're saying but I would like to understand. (I probably miss something)
In this case, for example,at line 70:
name is string containing "Legal Leaves 2021"
extra is string containing " (20 days available)"

record_name is string = name + " " + extra   = a string containing  "Legal Leaves 2021 (20 days available)"

So I don't understand how it can work in another way than having the full text "Legal Leaves 2021 (20 days available)" in the po file (or in the db)

Reply:
Pay attention to what's inside the `_()` and compare this

```
_('%(name)s (%(extra)s)') % {
    'name': 'A',
    'extra': 'B',
}
```

and 

```
_('%(name)s (%(extra)s)' % {
    'name': 'A',
    'extra': 'B',
})
```

Reply:
Ok, it allow to move vars in translations 
But in our case I think vars are not enough splitted to this use case.
Anyway I will undo this change.
Thanks for your time